legacy/requirements-cli.txt

```diff
@@ -0,0 +1,7 @@
+click
+numpy
+psutil
+py-cpuinfo
+pytest>=7.0.0
```

Comment:
I think this can be removed; I moved the new test dir outside of the package dir (which is a weird structure imo)